code/src/store/slices/userSlice.ts

```diff
@@ -3,6 +3,7 @@ import { sliceNames } from "@/store/constants";
 import { LOCAL_STORAGE_KEYS } from "@/constants/common";
 import { useAppSelector } from "@/hooks/use-redux/useRedux";
 import { UserDetails } from "@/types/user.types";
+import checkJWTTokenExpiration from "@/utils/check-jwt-token-expiration/checkJWTTokenExpiration";
```

Comment:
`JWT` it's json web token - don't repeat yourself in naming I mean `JWTToken` in `checkJWTTokenExpiration`

Reply:
Yeap, I understand. The reason I did this is because when our system will extend, new tokens can be introduced, for example resetPasswordToken, verifyEmailToken and so on. Adding jwt to name makes it super clear by intent